docs/integrations/unity_catalog.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Unity Catalog
+description: Accessing lakeFS-exported Delta Lake tables from Unity Catalog.
+parent: Integrations
+redirect_from: /using/unity_catalog.html
+---
+
+# Using lakeFS with the Unity Catalog
+
+{% include toc_2-3.html %}
+
+## Overview
+
+Databricks Unity Catalog serves as a centralized data governance platform for your data lakes.
+Through the Unity Catalog, you can search for and locate data assets across workspaces via a unified catalog.
+Leveraging the external tables feature within Unity Catalog, you can register a Delta Lake table exported from lakeFS and
+access it through the unified catalog. 
+The subsequent step-by-step guide will lead you through the process of configuring a [Lua hook]({% link howto/hooks/lua.md %})
+that exports Delta Lake tables from lakeFS, and subsequently registers them in Unity Catalog.
+
+## Prerequisites
+
+Before starting, ensure you have the following:
+
+1. Access to Unity Catalog
+2. An active lakeFS installation with S3 as the backing storage, and a repository in this installation.
```

Comment:
Let's document this also as a limitation. With our current support, exports won't work for Azure.